internal/processing/processors/v2alpha1/authorizationpolicy/creator.go

```diff
@@ -0,0 +1,184 @@
+package authorizationpolicy
+
+import (
+	"context"
+	"fmt"
+	gatewayv2alpha1 "github.com/kyma-project/api-gateway/apis/gateway/v2alpha1"
+
+	"github.com/kyma-project/api-gateway/internal/builders"
+	"github.com/kyma-project/api-gateway/internal/processing"
+	"github.com/kyma-project/api-gateway/internal/processing/hashbasedstate"
+	"istio.io/api/security/v1beta1"
+	securityv1beta1 "istio.io/client-go/pkg/apis/security/v1beta1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	audienceKey string = "request.auth.claims[aud]"
+)
+
+var (
+	defaultScopeKeys = []string{"request.auth.claims[scp]", "request.auth.claims[scope]", "request.auth.claims[scopes]"}
+)
+
+// Creator provides the creation of AuthorizationPolicy using the configuration in the given APIRule.
+// The key of the map is expected to be unique and comparable with the
```

Comment:
This is in place on the AuthorizationPolicyCreator interface since end of 2022: https://github.com/triffer/api-gateway/commit/e3912429bf4d7aca9ff5cb0a970d0f018a00ef61
At this time the function returned `map[string]*securityv1beta1.AuthorizationPolicy`. Now that we have a different return type, we can just remove the comment.